configs/common/packages.yaml

```diff
@@ -142,7 +142,7 @@
       # https://github.com/JCSDA/spack-stack/issues/769
       # also: ... extdata2g segfault UFS?
       version: ['2.40.3']
-      variants: ~shared ~extdata2g ~pflogger
+      variants: +shared +pflogger ~f2py
```

Comment:
Just a note that I think these are all the defaults of mapl. But probably good to keep them explicit until people get used to it. :)